polaris/polaris/templates/base.html

```diff
@@ -222,8 +222,8 @@
     </script>
 
     {# Rendered via Polaris javascript integrations #}
-    {% for script in scripts %}
-      {{ script|safe }}
+    {% for script_src in scripts %}
+      <script src="/static/{{ script_src }}"></script>
```

Comment:
The template logic will get more complex if we want to allow anchors to choose if the script should load synchronously or asynchronously. You'll likely have to iterate through a list of `TemplateScript` objects and check for an `async` property.